shared/profile/pgp/add.native.js

```diff
@@ -1,8 +1,7 @@
 // @flow
 import React, {Component} from 'react'
-import {Button, Input, PlatformIcon, SmallInput, StandardScreen, Text} from '../../common-adapters'
+import {Button, Input, KeyboardAvoidingBox, PlatformIcon, SmallInput, StandardScreen, Text} from '../../common-adapters/index.native'
```

Comment:
Any ideas on how to avoid the `.native` on the end of this import?  If I just remove it, then:
```
profile/pgp/add.native.js:3
  3: import {Button, Input, KeyboardAvoidingBox, PlatformIcon, SmallInput, StandardScreen, Text} from '../../common-adapters/index'
                            ^^^^^^^^^^^^^^^^^^^ Named import from module `../../common-adapters/index`. This module has no named export called `KeyboardAvoidingBox`.

Found 1 error
```